cmd/gotk/create_kustomization.go

```diff
@@ -111,6 +108,18 @@ func createKsCmdRun(cmd *cobra.Command, args []string) error {
 	if ksSource == "" {
 		return fmt.Errorf("source is required")
 	}
+
+	ksSourceKind := sourcev1.GitRepositoryKind
+	ksSourceName := ksSource
+	ksSourceElements := strings.Split(ksSource, "/")
```

Comment:
We are doing this in so many places now that I think it could be added to the utils.